writing-gleam/gleam-command.md

```diff
@@ -0,0 +1,78 @@
+---
+title: gleam command
+layout: page
+---
+
+The `gleam` command uses subcommands to access different parts of the functionality:
+
+## `new`
+
+`gleam new <name> [project-root]`
+
+Creates a folder with the necessary files for starting a new gleam project with the given `name`.
+The `project-root` defaults to the value of `name`.
+
+- `--description <description>`: Provide a description which is inserted to the `src/<name>.src.app`
+  file in the new project.
+- `--template <lib|app>`: Indicate whether to generate a project for a library (`lib`) or
+  application (`app`). Defaults to `lib`.
+
+## `build`
+
+`gleam build [project-root]`
+
+Builds the given gleam project. Defaults to the current directory.
```

Comment:
Can we add a note saying this command does not download hex packages for you, and that it is more commonly called indirectly by rebar3? This is often a source of confusion￼

Reply:
Done - I hope it reads reasonably.